### PR TITLE
[FaceAPI] Update Face samples to use detection_02 model

### PR DESCRIPTION
**Pull request by TFR258**
* Update multiple recognition models.
* Update multiple recognition model test.
* Update FaceSDK nuget package.
* Update faceSDK to Cognitive-Services-Face-WPF.
* Update FaceSDK to FaceApiSnapshotSample.

## API Name (Kind)
`Face`

## Purpose
Update the samples to use Face's latest detection model: detection_02

### CODE REVIEW

File: app-samples/FaceApiSnapshotSample/FaceApiSnapshotSample/Program.cs [hunk 1]
```diff
@@ -30,6 +30,11 @@ public class Sample
         /// </summary>
         private static readonly string recognitionModel = RecognitionModel.Recognition02;
 
+        /// <summary>
+        /// RecognitionModel for Face detection.
```
Comment (lebronJ) [inline app-samples/FaceApiSnapshotSample/FaceApiSnapshotSample/Program.cs]: `DetectionModel`

File: samples/Face/Verify.cs [hunk 2]
```diff
@@ -32,14 +33,16 @@ public static async Task Run(string endpoint, string key)
             List<DetectedFace> detectedFaces1 = await Common.DetectFaces(
                                                     client,
                                                     $"{ImageUrlPrefix}{sourceImageFileName1}",
-                                                    recognitionModel: recognitionModel);
```
Comment (lebronJ) [inline samples/Face/Verify.cs]: We should also add `detectionModel: detectionModel` on the 27th line's `Common.DetectFaces`

File: app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs [hunk 3]
```diff
@@ -305,7 +310,8 @@ private async void ImagePicker_Click(object sender, RoutedEventArgs e)
                                 FaceAttributeType.Occlusion,
                                 FaceAttributeType.Smile
                             },
-                            recognitionModel
+                            recognitionModel,
+                            detectionModel: detectionModel
```
Comment (TFR258) [inline app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/FaceDetectionPage.xaml.cs]: I need to remove this: ``detection_02`` doesn't support face landmarks

**Comment by TFR258** [comment]
@lebronJ , I ran a couple of tests on the code samples.
We are looking at negligible deltas (<+0.5% over 5 runs)

**Comment by TFR258** [comment]
The WPF is interesting to use. But I haven't notice any slow down

**Comment by lebronJ** [comment]
LGTM. Please also double check there is no accuracy drop for these samples, e.g. wrong identification result.

@yangyuan could you review and merge this? Thanks.

**Pull request merged**